scripts/typescript/normalize-import.spec.js

```diff
@@ -64,9 +64,9 @@ describe(`normalize-import`, () => {
   it(`should throw error if provided --output is not a valid directory`, () => {
     const utils = setup();
 
-    expect(() => execSync(`node ${utils.scriptPath} --output some/nonexistent/path`, { stdio: 'pipe' })).toThrowError(
-      `some/nonexistent/path doesn't exist`,
-    );
+    expect(() =>
+      execSync(`node ${utils.scriptPath} --output /some[\\/]nonexistent[\\/]path doesn't exist/`, { stdio: 'pipe' }),
```

Comment:
You can leave this line as it is; it's just the check for the error message that needs to change